src/main/java/ro/msg/learning/shop/controllers/CustomerController.java

```diff
@@ -0,0 +1,27 @@
+package ro.msg.learning.shop.controllers;
+
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import ro.msg.learning.shop.services.CustomerService;
+
+@RestController
+@CrossOrigin
+@AllArgsConstructor
+@RequestMapping("/customer")
+public class CustomerController {
+
+    private final CustomerService customerService;
+
+    @GetMapping("/{customerId}")
+    public ResponseEntity<Object> findById(@PathVariable("customerId") int customerId) {
+        return new ResponseEntity<>(customerService.findById(customerId), HttpStatus.OK);
```

Comment:
Many of these endpoints were not required, but you can leave them be.
Just be careful when retrieving customer data to not send the password or other sensitive details of the customer.